### DISCUSSION TITLE
get_block() functionality?

### DISCUSSION THREAD

**Discussion by bdp-utah1** [StefanoTrv/simple_icd_10_CM#6] [Q&A] [answered]
Hello,
I don't see any get_block() or equivalent method in the README. Am I missing something? If not, could this be added?
Thanks!

**Comment by StefanoTrv** [comment, marked as answer]
Hello,
I'll consider adding it in the future.
For now, you can use this simple implementation:
```
def get_block(code):
    ancestors = cm.get_ancestors(code)
    for c in ancestors:
        if cm.is_block(c):
            return c
    return None
```
